test/strings.jl

```diff
@@ -1447,3 +1447,6 @@ let s = "ba\0d"
     @test_throws ArgumentError Base.unsafe_convert(Cstring, s)
     @test_throws ArgumentError Base.unsafe_convert(Cwstring, wstring(s))
 end
+
+
+
```

Comment:
(blank lines?)

There should probably be a loop to test conversions for all UTF-xx types to all other UTF-xx types, for a variety of different strings (e.g. ASCII, codepoints only in BMP, codepoints outside of BMP, ...) to make sure that all the new code paths are exercised.